init plugin

nikpivkin opened the pull request.

nikpivkin committed:
infa: init dev environment

nikpivkin committed:
build: init plugin

nikpivkin committed:
add job template

nikpivkin committed:
build(dev): add a script to validate the report

nikpivkin committed:
add .env to gitignore

nikpivkin committed:
style: apply formatter

nikpivkin committed:
init go app

nikpivkin committed:
infa: fix docker config; add registry service

nikpivkin committed:
refactor: extract docker image from env

nikpivkin committed:
refactor: the plugin must run the go program

nikpivkin committed:
build: update Makefile

nikpivkin committed:
infa(dev): update docker-compose

nikpivkin committed:
doc: update readme

nikpivkin committed:
doc: add pipeline examples

nikpivkin committed:
update gitignore

nikpivkin committed:
doc: update examples

nikpivkin committed:
update doc; add template

nikpivkin committed:
build: update Makefile

nikpivkin committed:
doc: update readme

nikpivkin committed:
fix some bugs

nikpivkin committed:
build: remove entrypoint, build go for linux

nikpivkin committed:
add report-path and template-path flags

nikpivkin committed:
fix templates

afdesk commented on Makefile:
do we really need this Makefile here?

nikpivkin committed:
retrieve plugin and trivy versions

nikpivkin committed:
add template for dind

nikpivkin committed:
add example for cloud gl

afdesk commented on cmd/trivy-gitlab/main.go:
```diff
@@ -0,0 +1,74 @@
+		Use:   "",
+		Short: "",
+	}
+}
```
why do we need this function?

afdesk commented on go.mod:
```diff
@@ -0,0 +1,10 @@
+module github.com/afdesk/trivy-gitlab
+
+go 1.20
```
let's use 1.19

nikpivkin replied:
done

nikpivkin committed:
add more analyzers

nikpivkin committed:
move dev doc

nikpivkin committed:
container scanner must support version 15 of the schema

nikpivkin committed:
add tests

nikpivkin committed:
fix templates

nikpivkin committed:
fix: remove GIT_STRATEGY from file-dependent templates

nikpivkin committed:
refactor

Fs scanning must be performed with a single command.
Container scanning must be performed with a single command.
Instead of templates, use types from Trivy and Gitlab.

nikpivkin committed:
generate cyclonedx report

nikpivkin committed:
tests(fixture): add .env

nikpivkin committed:
extract commit with secret

nikpivkin committed:
resolve image name

nikpivkin committed:
use env var to select scanners

nikpivkin committed:
some fixes

nikpivkin committed:
doc: fix urls and examples

nikpivkin committed:
refactor: apply linter

nikpivkin committed:
refactor: improve error handling

nikpivkin committed:
build: use goreleaser

nikpivkin committed:
ci: add workflows

nikpivkin committed:
fix(ci): remove an extra step

nikpivkin committed:
chore: use latest release version

nikpivkin committed:
fix: ignore EINVAL

nikpivkin committed:
doc: update readme

nikpivkin committed:
fix log message

nikpivkin committed:
refactor: remove cobra

nikpivkin committed:
refactor: change the way versions are received

nikpivkin committed:
refactor: simplify

nikpivkin committed:
build: downgrade the go version to 1.19

nikpivkin committed:
refactor: mv to internal

nikpivkin committed:
ci: use reusable workflow

nikpivkin committed:
refactor: pass arguments trivy as is

nikpivkin committed:
chore: update plugin.yaml
